CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Change log
 
+## 5.0.0 (2018-06-18)
+
+Breaking changes: 
+
+* (Paul Sadauskas) Add support for --stdin-filename
```

Comment:
I wouldn't call this breaking :slightly_smiling_face:.

Reply:
Good catch, moved down.